SearchResults.js

```diff
@@ -102,42 +105,75 @@ function SearchResults() {
   self.loading = ko.pureComputed(function () { self.activeRequests() == 0; });
   self.uninitialised = ko.pureComputed(function () { return self.pullRequests().length == 0; });
 
+
+  function loadAllPages(query, onComplete) {
+    var totalCount;
+    var pullRequests = [];
+    // Function to load a page of results
+    function getPage(uri) {
+      self.activeRequests(self.activeRequests() + 1);
+      $.getJSON(uri, function (data, textStatus, jqXHR) {
+        self.errorMessage(null);
+        totalCount = data.total_count;
+        // Add the pull requests to the existing cache
+        pullRequests = pullRequests.concat(data.items);
+        // Get the link to the next page of results
+        var nextLinkSuffix = "; rel=\"next\"";
```

Comment:
You can use single quotes to avoid needing to backslash enclosed double quotes:

    var nextLinkSuffix = '; rel="next"';